docs/webengine-attributes.md

```diff
@@ -1,4 +1,4 @@
-> The official Splunk documentation for this page is [Webengine attributes](https://docs.splunk.com/Observability/gdi/get-data-in/application/java/configuration/java-otel-metrics-attributes.html#middleware-attributes-java-otel).
+> The official Splunk documentation for this page is [Middleware attributes](https://docs.splunk.com/Observability/gdi/get-data-in/application/java/configuration/java-otel-metrics-attributes.html#middleware-attributes-java-otel). For instructions on how to contribute to the docs, see [CONTRIBUTE.md](../CONTRIBUTE.md).
```

Comment:
Agent does not produce `middleware` attributes anymore, it produces `webengine`

Reply:
Fixed!